src/python/py_whisper_pipeline.cpp

```diff
@@ -410,6 +393,25 @@ void init_whisper_pipeline(py::module_& m) {
             "streamer",
             (whisper_generate_docstring + std::string(" \n ") + whisper_generation_config_docstring).c_str())
 
+        .def(
+            "generate",
+            [](WhisperPipeline& pipe,
+               const RawSpeechInput& raw_speech_input,
+               const WhisperGenerationConfig& generation_config,
+               const std::shared_ptr<ChunkStreamerBase>& streamer,
+               const py::kwargs& kwargs) -> py::typing::Union<ov::genai::WhisperDecodedResults> {
+                StreamerVariant _streamer = std::make_shared<ov::genai::ChunkToBaseStreamerAdapter>(streamer);
```

Comment:
maybe we can add deprecation warning using `PyErr_WarnEx(PyExc_DeprecationWarning .. ` ?

Reply:
Yes, will do